homework1/S3.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Scanner;
+public class S3 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        findEvenOdd();
+    }
+    public static void findEvenOdd() {
+        int N = 1000;
+        for(int i = 1; i < N; i++) {
+            if(i % 3 == 0)
```

Comment:
You are supposed to check if sum of digits is divisible by 3.

Reply:
Changed the code, which was expected